### ISSUE TITLE
[BUG] Reformatter adds second blank line

### ISSUE THREAD

**Issue by jayvdb** [Riverside-Healthcare/djLint#79]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Windows
 - Python Version: 3.7
 - djLint Version: 0.5.4

## Issue
I have `endcomment`  in `blank_line_after_tag`, and I end up with an additional blank line after the tag.

This is a fictitious snippet like the cause to give slightly more context and remove what are hopefully unrelated bits of syntax.
```diff
 <script>
     $(document).ready(function() {
         {% comment %}
         ...
         {% endcomment %}
 
+
     });
 </script>
```

## How To Reproduce
If more context is need, `common/templates/common/navigation.html` is where this is happening.

**Comment by christopherpickering** [comment]
I'm pretty sure this is limited to cases where there is an ignored block inside another ignored block. The issue is that the indenter processes line by line. I've update the other functions to process the code as a block... intent will be a project to update I think. It still needs to go line-by-line, but also need to be aware if it is inside an ignored block at the same time.

**Comment by christopherpickering** [comment]
Fixed with 507dae9
